topology/Fermi National Accelerator Laboratory/FermiGrid/FNAL_FERMIGRID_downtime.yaml

```diff
@@ -1,26 +1 @@
-- Class: SCHEDULED
-  Description: The whole infrastructure behind the CE is being redone. The CE will
-    need to be retired once the refactor is done. Putting it in downtime for now so
-    we can rollback quickly if needed.
-  EndTime: Oct 26, 2017 18:00 +0000
-  ID: 1005428
-  ResourceName: FNAL_GPGRID_4
-  Services:
-  - CE
-  - GridFtp
-  Severity: Outage
-  StartTime: Sep 18, 2017 14:00 +0000
-# ----------------------------------------------------------
-- Class: SCHEDULED
-  Description: The whole infrastructure behind the CE is being redone. The CE will
-    need to be retired once the refactor is done. Putting it in downtime for now so
-    we can rollback quickly if needed.
-  EndTime: Oct 26, 2017 18:00 +0000
-  ID: 1005429
-  ResourceName: FNAL_GPGRID_5
-  Services:
-  - CE
-  - GridFtp
-  Severity: Outage
-  StartTime: Sep 18, 2017 14:00 +0000
-# ----------------------------------------------------------
+
```

Comment:
Same with `topology/Fermi National Accelerator Laboratory/FermiGrid/FNAL_FERMIGRID_downtime.yaml` -- you left a single blank line, but we just need to remove the file.